go.mod

```diff
@@ -41,7 +41,6 @@ require (
 	github.com/ipfs/go-ipfs v0.4.22 // indirect
 	github.com/ipfs/go-ipfs-addr v0.0.1
 	github.com/ipfs/go-path v0.0.7
-	github.com/ipfs/go-peertaskqueue v0.0.5-0.20190704154349-f09820a0a5b6 // indirect
```

Comment:
Also at line 32 `github.com/gin-contrib/cors v1.3.0` shouldn't be imported anymore, right?

Reply:
yea, had to tidy some dependencies.